offload/libomptarget/OpenMP/OMPT/Callback.cpp

```diff
@@ -421,18 +421,17 @@ void Interface::beginTargetRegion() {
   // Set up task state
   assert(ompt_get_task_data_fn && "Calling a null task data function");
   TaskData = ompt_get_task_data_fn();
-  // Set up target task state
-  assert(ompt_get_target_task_data_fn &&
-         "Calling a null target task data function");
-  TargetTaskData = ompt_get_target_task_data_fn();
-  // Target state will be set later
-  TargetData = ompt_data_none;
+  // Set up target task and target state
+  assert(ompt_get_task_info_target_fn &&
+         "Calling a null target task info function");
+  if (ompt_task_info_t *TempTaskInfo = ompt_get_task_info_target_fn())
```

Comment:
Is `TempTaskInfo` a pointer to task info coming from libomp? If so, I think the name is misleading and it should be renamed to reflect that.